v2/stacks/auth/README.md

```diff
@@ -36,3 +36,35 @@ To get the aws values from the dashboard do the following:
 - copy the highlighted values
 
 Note that the AWS_SESSION_TOKEN is only valid for 12 hours. Once the token has expired you would need to stop the stack, retrieve and set new credentials before running the stack again.
+
+# Example on how to debug a Go application running in docker 
+
+This example will show how we can debug `dp-identity-api` when we run the auth stack.
+
+- Copy and paste the following code into the `Dockerfile.local` file in `dp-identity-api` repo
```

Comment:
Sorry, but it wasn't clear if you were adding to - or replacing - the file contents.

```suggestion
- Replace the contents of the `Dockerfile.local` file (in the `dp-identity-api` repo) with the following code
```

Reply:
This is now addressed.